content/unsupported.html

```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+title: Unsupported Browser Version
+description: Scratch Addons may not run successfully because your browser version is unsupported.
+---
+
+<section id="intro" class="intro-heading">
+  <h1 class="mb-3">Unsupported Browser Version!</h1>
+  <p class="mb-1">
+    Scratch Addons may not run successfully because your browser version is
+    unsupported.
+  </p>
+</section>
+
+<section class="container">
+  <p>Try reinstalling or updating your browser to run Scratch Addons</p>
+  <p>
+    <span class="iconify" data-icon="simple-icons:googlechrome"></span> Chromium 80+ (
+    <a href="https://google.com/chrome/"><span class="iconify" data-icon="simple-icons:googlechrome"></span> Chrome 80+</a>,
+    <a href="https://microsoft.com/en-us/edge/"><span class="iconify" data-icon="simple-icons:microsoftedge"></span> Microsoft Edge 80+</a>,
+    <a href="https://opera.com/download/"><span class="iconify" data-icon="simple-icons:opera"></span> Opera 67+</a>,
+    <a href="https://brave.com/download/"><span class="iconify" data-icon="simple-icons:brave"></span> Brave 1.3+</a>,
+    <a href="https://vivaldi.com/download/"><span class="iconify" data-icon="simple-icons:vivaldi"></span> Vivaldi 2.11+</a>)
```

Comment:
These should be site params

Reply:
Please explain... W_L said he wants all links here

Reply:
Use {{ .Site.Params.min_firefox_version }} and define that in the params sections of config.yml